### PR TITLE
Feature/public view home

### PR DESCRIPTION
**Pull request by minotogna**
Initial pull request: moved some components under common components folder

### CODE REVIEW

File: webapp/components/description/actions.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import axios from 'axios'
-import * as autosave from '../autosave/actions'
+import * as autosave from '../../autosave/actions'
```
Comment (sorja) [inline webapp/components/description/actions.js]: Possible to import using `@webapp`...?

File: webapp/components/description/style.less [hunk 2]
```diff
@@ -1,5 +1,5 @@
-@import (reference) "../app-styles/config";
-@import (reference) "../app-styles/buttons";
+@import (reference) "../../app-styles/config";
+@import (reference) "../../app-styles/buttons";
```
Comment (sorja) [inline webapp/components/description/style.less]: Is it possible to use relative imports in less?

File: webapp/main.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import './app-styles/style.less'
-import './ckEditor/style.less'
+import './components/ckEditor/style.less'
```
Comment (sorja) [inline webapp/main.js]: Possible to use `@webapp`..?

File: webapp/originalDataPoint/components/commentsEditor.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-import ckEditorConfig from '../../ckEditor/ckEditorConfig'
+import ckEditorConfig from '../../components/ckEditor/ckEditorConfig'
```
Comment (sorja) [inline webapp/originalDataPoint/components/commentsEditor.js]: Possible to use `@webapp`..?

File: webapp/components/description/style.less (hunk 2 above)
Comment (minotogna) [reply webapp/components/description/style.less]: yes

**Pull request merged**